_workinggroups/generators.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Physics Generators"
+layout: plain
+---
+
+The Physics Generators Working Group is a common forum for discussion and 
+technical work on the physics event generators used by HEP experiments.
+It promotes the collaboration of experimental and theoretical physicists from 
+different experiments and generator teams and of software and computing engineers,
+with the aim of having them work together on improving the current codes 
+and production workflows and on making new theoretical advances 
+easier to implement in a computationally efficient way.
+
+The working group was set up as a follow-up of the Physics Event Generator 
+Computing Workshop that was held at CERN in November 2018, during which 
+an initial program of work in this area was identified.
+The working group is currently coordinated by Andrea Valassi, Josh McFayden, 
+Stefan Hoeche, Steve Mrenna and Taylor Childers.
+
```

Comment:
Thanks Graeme. I added the mailing list.
I populated it with the list of workshop organisers for the moment.
I will then send an email via indico to all workshop participants, asking that they register if interested. I will do this after merging this MR, so that I can also point to a web page :-)
Andrea